server/accounts.go

```diff
@@ -2574,6 +2582,7 @@ func (s *Server) UpdateAccountClaims(a *Account, ac *jwt.AccountClaims) {
 		}
 	}
 	a.defaultPerms = buildPermissionsFromJwt(&ac.DefaultPermissions)
+	a.incomplete = incomplete
 	a.mu.Unlock()
```

Comment:
nit but maybe here check incomplete and do the ExporterMap assignment once here.

Reply:
Not sure I follow. When we iterated over all imports and did NOT run into an error, it does not matter what value a.incomplete had. At this point it is complete.

Since each import can refer to a different export there is not really such a thing as once.
I assume assigning exportermap once is about setting it together with a.incomplete. (will change that)